lib/lib.rs

```diff
@@ -243,6 +246,121 @@ impl Parser {
   }
 }
 
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct EszipError {
+  message: String,
+  specifier: Option<String>,
+  line: Option<usize>,
```

Comment:
Maybe add a doc saying what kind of indexes the line and column are. I believe the line is 1-indexed and column is 1-indexed character index (not byte)?